contracts/RibbonStakingRewards.sol

```diff
@@ -150,15 +150,15 @@ contract StakingRewards is
         require(amount > 0, "Cannot withdraw 0");
         _totalSupply = _totalSupply.sub(amount);
         _balances[msg.sender] = _balances[msg.sender].sub(amount);
-        if(block.timestamp < periodFinish.add(8 days)){
+        if(block.timestamp < periodFinish.add(1 days)){
           rewards[msg.sender] = 0;
         }
         stakingToken.safeTransfer(msg.sender, amount);
         emit Withdrawn(msg.sender, amount);
     }
 
     function getReward() public override nonReentrant updateReward(msg.sender) {
-        uint256 reward = block.timestamp >= periodFinish.add(8 days) ? rewards[msg.sender] : 0;
+        uint256 reward = block.timestamp >= periodFinish.add(1 days) ? rewards[msg.sender] : 0;
```

Comment:
can move this now to just 1 more day. if you deposit on third week, the lastTimeRewardApplicable will be periodFinish (because we shifted _numweekspassed up one). The lastTimeRewardApplicable will not change afterwards.